client/src/components/calendarDay.jsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { RegularDay } from '../styles/styles.jsx';
+import { AvailableDay } from '../styles/styles.jsx';
+import { EmptyDay } from '../styles/styles.jsx';
+import { Div } from '../styles/styles.jsx';
+
+class CalendarDay extends React.Component {
+  
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+    if( this.props.day && this.props.cost === undefined || this.props.cost === '' ){
+      return (
+        <RegularDay>
+          <Div>
+            {  this.props.day }
+          </Div>
+          <div>
+            {  this.props.cost === undefined || this.props.cost === '' ? '' : '$'+this.props.cost  }
+          </div>
+        </RegularDay>
+      )
+    } else if (this.props.day === undefined && this.props.cost === undefined || this.props.cost === '' ) {
+      return (
+        <EmptyDay>
+          <Div>
+            {  this.props.day }
+          </Div>
+          <div>
+            {  this.props.cost === undefined || this.props.cost === '' ? '' : '$'+this.props.cost  }
+          </div>
+        </EmptyDay>
+      )
+    } else {
+      return (
+        <AvailableDay>
```

Comment:
Since you are basically doing the same thing in these if/else statements, you can just reuse one styled component to do what you want it to do. The styled components have logic in them so that you can make them conditionally render different things.